frontend/src/component/ui/ChangeLogModal.scss

```diff
@@ -0,0 +1,113 @@
+@import '../styles/color';
+@import '../styles/corner';
+@import '../styles/scrollbar';
+
+.modal-wrapper {
+  left: 0%;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+
+  .modal-body {
+    background-color: $color-white;
+    border: 1px solid $color-light-gray-1;
+    left: 50%;
+    position: fixed;
+    top: 50%;
+    width: 500px;
+    height: 500px;
+    transform: translateX(-50%) translateY(-50%);
+    @include round-corner;
+
+    .modal-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      font-size: 20px;
+      line-height: 1;
+      text-align: left;
+      padding: 20px;
+      border-bottom: 1px solid $color-light-gray-1;
+      font-weight: 600;
+
+      i.clear {
+        cursor: pointer;
+      }
+    }
+
+    ul {
+      height: calc(500px - 114px);
+      list-style: none;
+      display: block;
+      padding: 0;
+      margin: 0;
+      overflow-y: scroll;
+      background-color: $color-white-smoke;
+      border: 4px solid $color-white-smoke;
+      @include scroll-bar;
+
+      li {
+        padding: 10px;
+        list-style: none;
+        text-align: left;
+        border: 1px solid $color-light-gray-1;
+        margin: 10px;
+        min-height: 120px;
+        background: white;
+        position: relative;
+        word-break: break-all;
+
+        .title {
+          font-weight: 900;
+          padding-bottom: 6px;
+        }
+
+        .published-date {
+          position: absolute;
+          bottom: 10px;
+          color: $color-light-gray-2;
+        }
+      }
+    }
+
+    ul.complete-list {
+      height: calc(500px - 60px);
+    }
```

Comment:
Increase height of list and remove "view all updates" button when the button is clicked.